ddtrace/contrib/azure.functions/patch.py

```diff
@@ -0,0 +1,39 @@
+from ddtrace import config
```

Comment:
Can you install hatch and then run the following command: `hatch run lint:fmt`.  This command will resolve the linting errors you're seeing in our tests